tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_i1.h

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "noc_nonblocking_api.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+#define POLYVAL10_DERIVATIVE(coef10, coef9, coef8, coef7, coef6, coef5, coef4, coef3, coef2, coef1, coef0, t4)  \
+    ((coef1 + \
```

Comment:
Please write a unit test for forward i1 if possible

Reply:
I haven't enabled pybind for this op yet since it is an intermediate op implemented only for i0_bw